tests/test_methods_nai.py

```diff
@@ -0,0 +1,58 @@
+"""This test aims to compare to previous PDA and PDAW results that were published and compared to exact QD.
+We use NaI as an example with two different laser pulses. If this test fails, the results produced by the code are unreliable!"""
+
+import numpy as np
+
+from promdens.promdens import InitialConditions, LaserPulse
+
+
+def test_pdaw_nai(make_pulse):
```

Comment:
```suggestion
def test_pdaw_nai():
```

`make_pulse` fixture is unused in this test, and also the test below.